paper.md

```diff
@@ -59,7 +59,8 @@ Two real-world use cases for `mpi4jax` are the ocean model Veros [@hafner_veros_
 
 The implementation of a primitive in `mpi4jax` consists of two parts:
 
-1. A Python module that registers a new primitive with JAX. JAX primitives consist of several parts, such as an *abstract evaluation* rule and several *translation rules*. The abstract evaluation rule is used by the compiler to infer output shapes and data types without running the actual computation, while translation rules supply the specific computational kernel to be called and prepare input buffers.
+1. A Python module, registering a new primitive with JAX. JAX primitives consist of an _abstract evaluation_ rule and several _translation rules_. The former is used by the compiler to infer the output shapes and data types without running the actual computation, while _translation rules_ determine the specific computational kernel and prepare the input buffers. A different _translation rule_ is necessary for every type of backend, such as CPUs, GPUs and TPUs.
+On specific primitives we also define the _transposition rule_ in order to support reverse mode Automatic Differentiation.
```

Comment:
This should come after the next paragraph, as it directly relates to the translation rules described before.